.jenkins/testing

```diff
@@ -1,17 +1,46 @@
 pipeline {
   agent { label 'docker' }
+  environment {
+    TESTING_CONTAINER_NAME = "cnx-func-testing-${env.BUILD_ID}"
```

Comment:
`BUILD_ID` isn't going to cut it, but I'll touch this up in a followup PR. The `BUILD_ID` is just a number, so it's possible that things could eventually collide, since the numbers are sequential by branch and pr, all starting at `1`.